packages/ui-react/src/pages/ScreenRouting/mediaScreens/MediaStaticPage/MediaStaticPage.tsx

```diff
@@ -16,6 +16,10 @@ const MediaStaticPage: ScreenComponent<PlaylistItem> = ({ data }) => {
   const pageTitle = `${data.title} - ${siteName}`;
   const canonicalUrl = data ? `${window.location.origin}${mediaURL({ media: data })}` : window.location.href;
 
+  useEffect(() => {
+    (document.scrollingElement || document.body).scroll({ top: 0 });
+  }, [data]);
```

Comment:
Maybe a dependecy on `data.feedid` is more robust?
```suggestion
  }, [data.feedid]);
```

Reply:
`data.mediaid` rather (it exists right?), because you could theoretically switch static page having the same `feedid`.